### PR TITLE
fix VersionUtility tests

### PR DESCRIPTION
**Pull request by anurse**

### CODE REVIEW

File: test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs [hunk 1]
```diff
@@ -7,48 +7,55 @@ namespace Microsoft.Framework.Runtime.Tests
     public class VersionUtilityFacts
     {
         [Theory]
-        [InlineData("net45", "dnx451", false)]
+        [InlineData("net45", "aspnet50", false)]
```
Comment (anurse) [inline test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs]: This is restoring the old test values. Eventually they can be dnxified but for now it's sufficient to add the below cases

File: test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs [hunk 2]
```diff
@@ -7,48 +7,55 @@ namespace Microsoft.Framework.Runtime.Tests
...
         [InlineData("dnxcore50", "portable-net45+win8", true)]
         [InlineData("dnxcore50", "portable-net451+win81", true)]
         [InlineData("dnxcore50", "portable-net40+sl5+win8", false)]
         [InlineData("dnxcore50", "portable-net45+win8", true)]
         [InlineData("dnxcore50", "portable-net451+win81", true)]
         [InlineData("dnxcore50", "portable-net40+sl5+win8", false)]
-
-        // Tests for aspnet -> dnx rename
-        // DNX packages installed into DNX451 projects? OK!
-        [InlineData("dnx451", "dnx452", true)]
```
Comment (davidfowl) [inline test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs]: bring this back

Comment (anurse) [inline test/Microsoft.Framework.Runtime.Tests/VersionUtilityFacts.cs]: As discussed in JabbR, this was a bad test. They should be swapped. I'll add that in to the final merge.

**Comment by davidfowl** [comment]
:shipit:

**Pull request closed without merging**